Eatery Blue/UserDefaultsKeys.swift

```diff
@@ -10,6 +10,7 @@ import Foundation
 enum UserDefaultsKeys {
 
     static let didOnboard = "didOnboard"
+    static let didOnboardCompareMenus = "didOnboardCompareMenus"
```

Comment:
I believe the Figma design had more onboarding screens for comparing menus.

Reply:
yeah i forgot one oops